CMakeLists.txt

```diff
@@ -225,10 +226,10 @@ if(NETWORKIT_PYTHON)
 	# DSOs on Apple OSes use different conventions for RPATH.
 	if(APPLE)
 		set_target_properties(_NetworKit PROPERTIES
-				INSTALL_RPATH "@loader_path")
+				INSTALL_RPATH "@loader_path;${NETWORKIT_RPATH}")
```

Comment:
I think the rpath given as a command line parameter should overwrite the entire rpath. If the user wants to keep `@loader_path`/`$ORIGIN`, they can pass an appropriate rpath.